enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -112,7 +112,8 @@ func (i *InvocationStatService) getTrendBasicQuery(timezoneOffsetMinutes int32)
 	    SUM(total_download_size_bytes) as total_download_size_bytes,
 	    SUM(total_upload_size_bytes) as total_upload_size_bytes,
 	    SUM(total_download_usec) as total_download_usec,
-        SUM(total_upload_usec) as total_upload_usec
+        SUM(total_upload_usec) as total_upload_usec,
+		SUM(total_cached_action_exec_usec) as total_cpu_micros_saved
```

Comment:
ahaha i don't know what's going on with the indentation here, but can you make this match the previous lines?

Reply:
uhhh tabs..Fixed

Reply:
ahahaha i don't want to keep nitpicking this but these last 3 lines are still misaligned relative to the lines above it and i assume your editor is just aligning to whatever came on the line before--
![image](https://user-images.githubusercontent.com/10570856/235011283-9ac844e9-e310-4597-a342-859aa4405be5.png)